plugins/inputs/intel_pmu/intel_pmu.go

```diff
@@ -270,11 +271,16 @@ func (i *IntelPMU) Stop() {
 	}
 }
 
-func newReader(files []string) (*ia.JSONFilesReader, error) {
+func newReader(log telegraf.Logger, files []string) (*ia.JSONFilesReader, error) {
 	reader := ia.NewFilesReader()
 	for _, file := range files {
 		err := reader.AddFiles(file)
 		if err != nil {
+			var deprecatedFormatError *ia.DeprecatedFormatError
+			if errors.As(err, &deprecatedFormatError) {
+				log.Warnf("%v. Consider using perfmon event file in updated format", deprecatedFormatError)
```

Comment:
Hi,

What does this full message look like?

Reply:
Hi! Example of the log will look like:

> deprecated JSON format was used while reading following files: /home/ubuntu/telegraf-bin/old-events-format.json. Consider using perfmon event file in updated format

Reply:
Would the user know where to get the new version of the files? Is that just the perfmon repo itself?

Reply:
Yes, exactly, it's just perfmon repo itself, and user can find the link in README

Reply:
Can we make that very clear:

```diff
- Consider using perfmon event file in updated format
+ See the perfmon repo for updated event files

Reply:
Done